### PR TITLE
add RasterTilesViz

### PR DESCRIPTION
**Pull request by vincentsarago**
This PR adds a simple `raster-tiles` visualization.

capture d ecran 2018-03-08 a 10 09 44

### CODE REVIEW

**Comment by akacarlyann** [comment]
Sweet! I'd really been hoping someone would hit this :)

**Comment by vincentsarago** [comment]
@ryanbaumann I did some edits + added docs.
you can have an other 👀 and merge it if it's 👌

File: mapboxgl/templates/raster.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
 document.getElementById('legend').style.visibility = 'hidden';
 
 map.on('style.load', function() {
-
+  console.log('Yoooooo');
```
Comment (ryanbaumann) [inline mapboxgl/templates/raster.html]: We could remove this line ;)

Comment (vincentsarago) [reply mapboxgl/templates/raster.html]: oups!

**Comment by vincentsarago** [comment]
@ryanbaumann `matplotlib` is a preste standard library when working with Jupyter and image data. we don't just use `imread` but also `imsave`. I'm eager to keep it as it is, plus it makes really easy to apply colormap.

**Comment by ryanbaumann** [comment]
Thanks, @vincentsarago.  Agree, most always have `matplotlib` installed from prior work or via Anaconda as is.

**Pull request merged**